packages/flutter_localizations/lib/src/l10n/cupertino_en.arb

```diff
@@ -0,0 +1,77 @@
+{
+  "datePickerHourSemanticsLabelOther": "$hour o'clock",
+  "@datePickerHourSemanticsLabel": {
+    "description": "Accessibility announcement for the selected hour on a time picker such as '5 o'clock' or '5点'",
+    "plural": "hour"
+  },
+
+  "datePickerMinuteSemanticsOne": "1 minute",
+  "datePickerMinuteSemanticsOther": "$minute minutes",
+  "@datePickerMinuteSemanticsLabel": {
+    "description": "Accessibility announcement for the selected minute on a time picker such as '15 minutes' or '15分'",
+    "plural": "minute"
+  },
+
+  "datePickerDateOrder": "mdy",
```

Comment:
Do we really use abbreviations in enums like this? Flutter is typically monthDayYear

Reply:
This is referring to https://github.com/xster/flutter/blob/9da3df5ba4e4cac46620e153bdf972ebde25bd58/packages/flutter/lib/src/cupertino/localizations.dart#L34. What were you referring to in Flutter as typical? Maybe we shouldn't have created that enum if an existing one's use is already widespread... Though that's a breaking API>

Reply:
I didn't mean that there was an existing enum with similar semantics, I was just surprised by the abbreviation. Water under the bridge now :-)